### PR TITLE
Add org.gnome.Games.LibretroPlugin.PicoDrive, take 2

### PR DESCRIPTION
**Pull request by Exalm**
Taking over https://github.com/flathub/flathub/pull/63

### CODE REVIEW

**Comment by Exalm** [comment]
bot, build

**Comment by Exalm** [comment]
bot, build org.gnome.Games.LibretroPlugin.PicoDrive

**Comment by Exalm** [comment]
bot, build org.gnome.Games.LibretroPlugin.PicoDrive

**Comment by Exalm** [comment]
bot, build org.gnome.Games.LibretroPlugin.PicoDrivebot, build org.gnome.Games.LibretroPlugin.PicoDrive

**Comment by Exalm** [comment]
bot, build org.gnome.Games.LibretroPlugin.PicoDrive

File: org.gnome.Games.LibretroPlugin.PicoDrive.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Adrien Plazas -->
+<component type="addon">
+  <id>org.gnome.Games.LibretroPlugin.PicoDrive</id>
+  <extends>org.gnome.Games.desktop</extends>
```
Comment (bilelmoussaoui) [inline org.gnome.Games.LibretroPlugin.PicoDrive.metainfo.xml]:
```suggestion
  <extends>org.gnome.Games</extends>
```
Should be the id of the app that it extends

Comment (bilelmoussaoui) [reply org.gnome.Games.LibretroPlugin.PicoDrive.metainfo.xml]: Unless Games uses that as an id in the metainfo

File: org.gnome.Games.LibretroPlugin.PicoDrive.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    "build-extension": true,
+    "appstream-compose": false,
+    "finish-args": [
+        /* Needed to find the ROMs */
+        "--filesystem=home:ro"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g"
+    },
```
Comment (bilelmoussaoui) [inline org.gnome.Games.LibretroPlugin.PicoDrive.json]: These options are set by default

File: flathub.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": [ "i386", "x86_64", "aarch64" ]
```
Comment (bilelmoussaoui) [inline flathub.json]: i386 isn't supported anymore

File: org.gnome.Games.LibretroPlugin.PicoDrive.metainfo.xml (hunk 1 above)
Comment (Exalm) [reply org.gnome.Games.LibretroPlugin.PicoDrive.metainfo.xml]: Nope, no .desktop there.

File: flathub.json (hunk 3 above)
Comment (Exalm) [reply flathub.json]: Whoops, I copied that from Games repo 👀️

**Comment by Exalm** [comment]
bot, build org.gnome.Games.LibretroPlugin.PicoDrive

**Comment by Exalm** [comment]
bot, build org.gnome.Games.LibretroPlugin.PicoDrive

File: org.gnome.Games.LibretroPlugin.PicoDrive.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        /* Needed to find the ROMs */
+        "--filesystem=home:ro"
+    ],
+    "cleanup": ["/include", "/lib/pkgconfig",
+                "/share/pkgconfig", "/share/aclocal",
+                "/man", "/share/man", "/share/gtk-doc",
+                "*.la", "*.a",
+                "/lib/girepository-1.0",
+                "/share/dbus-1", "/share/doc", "/share/gir-1.0"
+    ],
```
Comment (bilelmoussaoui) [inline org.gnome.Games.LibretroPlugin.PicoDrive.json]: Nitpick: I'm sure you don't need most of these

Comment (Exalm) [reply org.gnome.Games.LibretroPlugin.PicoDrive.json]: Right. None of them, in fact.

**Comment by Exalm** [comment]
bot, build org.gnome.Games.LibretroPlugin.PicoDrive

**Pull request closed without merging**